extensions/authy/README.md

```diff
@@ -0,0 +1,26 @@
+# Raycast Authy integration
+
+----------
+
+Warning: This extension is using Authy private api and may not compliant with the Terms of Service of Authy. Use at your own risk.
```

Comment:
Note about risk as requested here https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration